src/pages/aura.f7.html

```diff
@@ -174,61 +173,80 @@
           if (responseInProgress) return;
           // Receive dummy message
           receiveMessage(text);
-        });
+        });       
 
-        var answers;
+        function receiveMessage(received) {
+          var answers;
+          app.request.json('static/example-data/aura.json', function (data) {
+            answers = self.chat;
 
-        app.request.json('static/example-data/aura.json', function (data) {
-          answers = self.chat.answers;
-        });        
+            responseInProgress = true;
+            setTimeout(function () {
+              if(received.toLowerCase().search('acc') != -1){
+                var accs;
+                var answer = new Object();
+                answer.name = "Aura";
+                answer.avatar = "https://img.icons8.com/plasticine/2x/bot.png";
+                answer.type = "received";
 
-        function receiveMessage(received) {
-          responseInProgress = true;
-          setTimeout(function () {
-            if(received.toLowerCase().search('acc') != -1){
-              var accs;
-              var answer = new Object();
-              answer.name = "Aura";
-              answer.avatar = "https://img.icons8.com/plasticine/2x/bot.png";
-              answer.type = "received";
-
-              self.getACCHours().then(function (res) {
-                accs = res;
-                if(accs.length == 0){
-                  answer.text = "Não foi encontrada nenhuma ACC";
-                }
-                else{
-                  answer.text = '<b>'+accs.validated+'</b><br><br>';
-                  for(let i in accs.list){
-                    answer.text += '<b>Título</b>: '+accs.list[i].title+'<br>';
-                    answer.text += '<b>Data</b>: '+accs.list[i].date+'<br>';
-                    answer.text += '<b>Horas Validadas</b>: '+accs.list[i].hours+'<br>';
-                    answer.text += '<br>';
+                self.getACCHours().then(function (res) {
+                  accs = res;
+                  if(accs.length == 0){
+                    answer.text = "Não foi encontrada nenhuma ACC";
                   }
-                }
-              });
-            }
-            else{
-              // Get random answer
-              var answer = answers[Math.floor(Math.random() * answers.length)];
-            }
-            answer.textFooter = self.getTimestamp();
+                  else{
+                    answer.text = 'Isso é apenas um exemplo 🙂<br><b>'+accs.validated+'</b><br><br>';
+                    for(let i in accs.list){
+                      answer.text += '<b>Título</b>: '+accs.list[i].title+'<br>';
```

Comment:
Se for concatenar várias vezes algo em uma mesma variável, é mais legível fazer uso da variável uma unica vez. Uma sugestão de implementação seria:

```js
answer.text += '<b>Título</b>: '+accs.list[i].title+'<br>' +
                         '<b>Data</b>: '+accs.list[i].date+'<br>' +
                         '<b>Horas Validadas</b>: '+accs.list[i].hours+'<br>' +
                         '<br>';
```